cfpb/cfgov-refresh
# Updating some npm dependencies

Updating some npm dependencies

## Changes

- Modified `package.json` to update `webpack`, `del`,  `cucumber`, and `protractor-cucumber-framework`. 


## Testing

1. Run `gulp clean` and verify that 'static_built` is empty.
2. Run `gulp build` and verify that bundles are generated correctly ( some might be a little smaller) . 
3 Run ` gulp test:acceptance --fast`.

## Notes

-

## Todos

-

## Checklist
